test/WebJobs.Script.Tests/WorkerFunctionMetadataProviderTests.cs

```diff
@@ -180,5 +196,19 @@ public async void ValidateFunctionMetadata_Logging()
             Assert.Equal("Reading functions metadata (Worker)", traces[1].FormattedMessage);
             // The third log is Host is running without any initialized channels, restarting the JobHost. This is not relevant to this test.
         }
+
+        [Fact]
+        public void ValidateFunctionMetadata_IsoStringNotAltered()
+        {
+            FunctionMetadata functionMetadata = new FunctionMetadata();
+            List<string> rawBindings = new List<string>();
+            var isoString = "2025-02-10T22:45:33Z";
+            rawBindings.Add("{\"type\": \"cosmosDBTrigger\",\"name\": \"cosmosTrigger\",\"direction\": \"in\",\"databaseName\":\"databaseName\"," +
```

Comment:
May be raw literal string can be used to store the JSON and can avoid these many escape characters.